snap/hooks/install

```diff
@@ -30,16 +31,29 @@ fi
 chown -R 584788:root "${SNAP_COMMON}"/*
 
 # Copy over the mongod.conf and create needed directories/files
-echo "configuration file does not exist."
+echo "mongod configuration file does not exist."
 echo "copying default config to ${MONGO_CONFIG_FILE}"
 cp -r ${SNAP}/etc/mongod.conf ${MONGO_CONFIG_FILE}
 
+# Copy over the mongos.conf and create needed directories/files
+echo "mongos configuration file does not exist."
+echo "copying default config to ${MONGOS_CONFIG_FILE}"
+cp -r ${SNAP}/etc/mongod.conf ${MONGOS_CONFIG_FILE}
+
+
 
 # mongod.conf default values are not consistent with the snap directory system.
-sed -i "s/fork: true/fork: false/g" $MONGO_CONFIG_FILE
-sed -i "s:/var/log/mongodb:$LOGS:g" $MONGO_CONFIG_FILE
-sed -i "s:/var/lib/mongodb:$DATA:g" $MONGO_CONFIG_FILE
-sed -i "s:/var/run:/tmp:g" $MONGO_CONFIG_FILE
+yq -i ".processManagement.fork = false" $MONGO_CONFIG_FILE
+yq -i ".systemLog.path = \"${LOGS}/mongodb.log\"" $MONGO_CONFIG_FILE
+yq -i ".storage.dbPath = \"${DATA}\"" $MONGO_CONFIG_FILE
+yq -i ".processManagement.pidFilePath = \"/tmp/mongod.pid\"" $MONGO_CONFIG_FILE
+
+# mongos.conf default values are not consistent with the snap directory system.
+yq -i ".processManagement.fork = false" $MONGOS_CONFIG_FILE
+yq -i ".systemLog.path = \"${LOGS}/mongodb.log\"" $MONGOS_CONFIG_FILE
```

Comment:
I think this should be `mongod.log` - could you confirm?

Reply:
Confirmed, sorry about that